Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -339,6 +339,36 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("closePath")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddAction("ClearBetweenFrames",
+                _("Clear between frames"),
+                _("Clear the renderer before the next draw"),
```

Comment:
Activate (or deactivate) the clearing of the rendered shape at the beginning of each frame.